deploy/examples/create-external-cluster-resources.py

```diff
@@ -1219,7 +1214,30 @@ def create_rgw_admin_ops_user(self):
                     f"Code: {execErr.returncode}. Error: {execErr.stderr}"
                 )
                 sys.stderr.write(err_msg)
-                return None, None, False, "-1"
+            return info_cap_supported, None
+
+    def create_rgw_admin_ops_user(self):
+        if self._arg_parser.dry_run:
+            return self.create_rgw_user()
+
+        user_info = self.create_rgw_user()
```

Comment:
I think so `get_rgw_user` should be called from here only,

TO make it more clear

Reply:
okay